api/envoy/config/route/v3/route_components.proto

```diff
@@ -964,11 +975,18 @@ message RouteAction {
   // route.
   repeated RateLimit rate_limits = 13;
 
-  // Specifies if the rate limit filter should include the virtual host rate
-  // limits. By default, if the route configured rate limits, the virtual host
-  // :ref:`rate_limits <envoy_api_field_config.route.v3.VirtualHost.rate_limits>` are not applied to the
-  // request.
-  google.protobuf.BoolValue include_vh_rate_limits = 14;
+  oneof vh_rate_limits_options {
```

Comment:
Make a field in to `oneof` breaks the API policy as it breaks generated code. You can do this verification in code, or just make vh_rate_limits take precedence.